provisioning/provisioning-service-client/src/main/java/com/microsoft/azure/sdk/iot/provisioning/service/configs/X509CertificateInfo.java

```diff
@@ -42,50 +44,66 @@
     private static final String SUBJECT_NAME_TAG = "subjectName";
     @Expose
     @SerializedName(SUBJECT_NAME_TAG)
+    @Getter
+    @Setter
     private String subjectName;
 
     // the sha1 thumbprint of the X509 certificate
     private static final String SHA1_THUMBPRINT_TAG = "sha1Thumbprint";
     @Expose
     @SerializedName(SHA1_THUMBPRINT_TAG)
+    @Getter
+    @Setter
     private String sha1Thumbprint;
 
     // the sha256 thumbprint of the X509 certificate
     private static final String SHA256_THUMBPRINT_TAG = "sha256Thumbprint";
     @Expose
     @SerializedName(SHA256_THUMBPRINT_TAG)
+    @Getter
+    @Setter
```

Comment:
a couple of these need the @NotNull tag

Reply:
The service will validate this info for us